tests/run-make-cargo/build-std-configurations/rmake.rs

```diff
@@ -0,0 +1,124 @@
+// This test ensures we are able to compile -Zbuild-std=core under a variety of profiles.
+
+#![deny(warnings)]
+
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+use std::thread;
+
+use run_make_support::serde_json::{self, Value};
+use run_make_support::tempfile::TempDir;
+use run_make_support::{cargo, rfs, rustc};
+
+#[derive(Clone)]
+struct Task {
+    target: String,
+    opt_level: u8,
+    debug: u8,
+    panic: &'static str,
+}
+
+fn manifest(task: &Task) -> String {
+    let Task { opt_level, debug, panic, target: _ } = task;
+    format!(
+        r#"[package]
+name = "scratch"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Any reason for using 2021 rather than the current latest (2024)?

Reply:
Nope. The latest one is probably a sensible choice.